modules/reference/pages/data-transforms/js/js-sdk.adoc

```diff
@@ -0,0 +1,137 @@
+= JavaScript API for Data Transforms
+:description: Work with data transforms using JavaScript.
+:page-aliases: reference:data-transforms/js-sdk.adoc
+
+This page contains the API reference for the data transforms client library of the JavaScript SDK.
+
+== Functions
+
+[[OnRecordWritten]]
+=== onRecordWritten()
+
+  onRecordWritten (`cb`): `void`
+
+Registers a callback to be fired when a record is written to the input topic. This callback is triggered after the record has been written, fsynced to disk, and acknowledged by the producer. This method should be called in your script's entry point.
+
+==== Parameters
+
```

Comment:
Should this be Properties?

Reply:
Parameters is correct when describing what you can pass to functions.